src/main/java/com/google/api/gax/grpc/ApiCallSettings.java

```diff
@@ -44,44 +44,70 @@
  */
 public class ApiCallSettings {
 
-  private Set<Status.Code> retryableCodes = new HashSet<>();
-  private RetryParams retryParams = null;
+  protected ImmutableSet<Status.Code> retryableCodes;
```

Comment:
We don't have a need for any members in immutable classes to be protected; make them private.